### DISCUSSION TITLE
🐞

### DISCUSSION THREAD

**Discussion by justinthoms** [aldinokemal/go-whatsapp-web-multidevice#164] [General]
### What happened?

curl \
 -X POST http://localhost:3000/send/message \
 -H "Content-Type: multipart/form-data" \
 -F "phone=6289685028129@s.whatsapp.net" \
 -F "message=selamat\n malam" \
 -F "reply_message_id=3EB089B9D6ADD58153C561"

here the not enter in new line it get output like selamat\n malam whaen try \n\n its also same 

![image](https://github.com/aldinokemal/go-whatsapp-web-multidevice/assets/57831354/6d4524f1-c9f5-4d45-96b4-123831aa0dfc)

### Reproduction Steps

```bash
1.how
2.
3.
...
```

### Application Version

4.1.41

### Verification

- [X] I have checked for existing closed issues and discussions.

**Comment by aldinokemal** [comment]
you can try this

```bash
curl 'http://localhost:3000/send/message' \
  -H 'Accept: application/json, text/plain, */*' \
  -H 'Content-Type: application/json' \
  --data-raw '{"phone":"6289685028129@s.whatsapp.net","message":"ok\naman\ngood multiline"}'
```
